static/robots.txt

```diff
@@ -0,0 +1,4 @@
+User-agent: *
+Allow: /
+Allow: /account*
+Allow: /^[a-z0-9-]*[a-z][a-z0-9-]*$
```

Comment:
I'm not sure if regexp are allowed by standard, although it's possible that google/bing would recognise that.

On the other hand, I think `Allow: /` is enough to allow anything, so we don't need to specify snap details URL specifically.

Reply:
I don't really know I took it from Celso's comment I'll check this